src/SignhostAPIClient/Rest/ErrorHandling/BadGatewayException.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace Signhost.APIClient.Rest.ErrorHandling
+{
+	[Serializable]
+	public class BadGatewayException
+		: SignhostRestApiClientException
+	{
+		public BadGatewayException()
+			 : base()
+		{
+		}
+
+		public BadGatewayException(string message)
+			: base(message)
+		{
+		}
+
+		public BadGatewayException(string message, Exception innerException)
+			: base(message, innerException)
+		{
+			HelpLink = "https://api.signhost.com/Help";
+		}
+
+		protected BadGatewayException(SerializationInfo info, StreamingContext context)
+			: base(info, context)
+		{
+		}
+	}
+}
```

Comment:
Do we really want a specific error mesage for all http codes? Or should we accept the generic SignhostRestApiClientException in this case?